### ISSUE TITLE
Add demographic data loading

### ISSUE THREAD

**Issue by fccoelho** [AlertaDengue/PySUS#29]
Add a function to automatically download demographic data from IBGE, age-structured census, and projections.

**Comment by gabrielmcf** [comment]
This project may help you: https://github.com/renanbirck/pyibge

**Comment by fccoelho** [comment]
Yes, it is promising, as it wraps the SIDRA API. I have to take closer look at what is available through Sidra because it has the potential to be much more efficient to download than the geoftp.

**Comment by fccoelho** [comment]
This other SIDRA client looks more promising https://github.com/AlanTaranti/sidrapy

**Comment by fccoelho** [comment]
None of them were  well developed, so I added my own wrapper in  the [stats.py](https://github.com/AlertaDengue/PySUS/blob/master/pysus/demography/stats.py) module
